### PR TITLE
I was not using the batching so folks were getting spammed!

### PR DESCRIPTION
**Pull request by ddellacosta**

### CODE REVIEW

File: src/election_notification_works/handlers/voter_guide.clj [hunk 1]
```diff
@@ -99,12 +99,14 @@
   ([skip-hostnames user-batch-size {:keys [election-id] :as message}]
    (log/debug "election-notification-works.voter-guide/handle-election " (pr-str message))
    (let [election (:election (async/<!! (read-election {:id election-id})))
-         user-ids (:user-ids (async/<!! (read-election-uids {:election-id election-id})))
+         user-ids (->> (read-election-uids {:election-id election-id})
+                       async/<!!
```
Comment (danielglauser) [inline src/election_notification_works/handlers/voter_guide.clj]: Any need for a timeout here?

File: src/election_notification_works/handlers/voter_guide.clj [hunk 2]
```diff
@@ -99,12 +99,14 @@
...
+                       :user-ids)
          api-cache (atom (cache/ttl-cache-factory {} :ttl 100000))]
      (if (seq user-ids)
        (doseq [user-id-batch (partition-all user-batch-size user-ids)]
          (future
-           (let [user-batch (->> (async/<!! (page-user user-ids))
+           (let [user-batch (->> (async/<!! (page-user user-id-batch))
```
Comment (danielglauser) [inline src/election_notification_works/handlers/voter_guide.clj]: Diddo above, should we roll in some type of `take-or-timeout` like semantics?

File: test/election_notification_works/handlers/voter_guide_test.clj [hunk 3]
```diff
@@ -46,78 +51,96 @@
...
+                      (helpers/take-or-timeout channels/voter-guide-notify-ext 250)
+                      [:message :notification])))]
+
+        (is (= (into #{} (keep #(get-in % [:recipient :full-name]) notys))
+               #{"Jane Smith"}))
+
+        (is (= (into #{} (keep #(get-in % [:recipient :hostname]) notys))
+               #{"jane.turbovote.org"}))
+
+        ))))
```
Comment (danielglauser) [inline test/election_notification_works/handlers/voter_guide_test.clj]: Dangling parens.

File: test/election_notification_works/handlers/voter_guide_test.clj [hunk 4]
```diff
@@ -46,78 +51,96 @@
...
 
-      ;; WHY CAN'T I CALL THIS WITHOUT LOCKING UP THE TESTS??
-      ;; if both this and the above test of this handler is
-      ;; uncommented the tests freeze. They both pass when
-      ;; run independently, however.
-      #_
-      (testing "Skips hostnames"
-        (vg/handle-election
-         #{"bob.turbovote.org"} {:election-id (java.util.UUID/randomUUID)})
+          )))))
```
Comment (danielglauser) [inline test/election_notification_works/handlers/voter_guide_test.clj]: Dangling parens

File: src/election_notification_works/handlers/voter_guide.clj (hunk 1 above)
Comment (danielglauser) [inline src/election_notification_works/handlers/voter_guide.clj]: Forgot that Kehaaaar will handle the timeouts. Should we check for `nil`?

File: src/election_notification_works/handlers/voter_guide.clj (hunk 2 above)
Comment (danielglauser) [inline src/election_notification_works/handlers/voter_guide.clj]: Forgot that Kehaaaar will handle the timeouts. Should we check for nil?

Comment (ddellacosta) [inline src/election_notification_works/handlers/voter_guide.clj]: The `loop` semantics will take care of this.

**Pull request merged**